examples/apps/ai-collab/src/components/DiffViewer.tsx

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Diff } from "@fluidframework/ai-collab/alpha";
+import { Card, Typography, Box } from "@mui/material";
+import React from "react";
+
+interface DiffViewerProps {
+	diffs: Diff[];
+}
+
+export const DiffViewer: React.FC<DiffViewerProps> = ({ diffs }) => {
```

Comment:
Docs please :)

Reply:
It's just for local testing. I will remove this react component :)